prow-configurations/README.md

```diff
@@ -31,7 +41,17 @@ The installation script performs the following steps to install Prow:
 - Create secrets for HMAC and OAuth2 to be used by Prow.
 - Deploy Prow components with the `a202e595a33ac92ab503f913f2d710efabd3de21`revision.
 - Add annotations for the Prow Ingress to make it work with the NGINX Ingress Controller.
+<!--- 
+TODO not implemented yet:
 - Upload the set of configurations for plugins.
+-->
+
+To check if installation was successful, perform following steps:
+- check if all pods are up and running:
```

Comment:
I assume that the order of those steps is essential? If yes, I'd introduce an ordered list:
`1.  Check if all Pods are up and running:`
2. ...